ui/src/tests/Product.test.tsx

```diff
@@ -316,7 +316,7 @@ describe('Products', () => {
                 const tagsLabelElement = await app.findByLabelText('Product Tags');
                 const containerToCreateFinTech = {
                     container: await app.findByTestId('productForm'),
-                    createOptionText: 'Press Enter to add "Fin Tech"',
+                    createOptionText: 'Create "Fin Tech"',
```

Comment:
Why use the TestUtils.expectedCreateOptionText function in the previous test (line 285 in this PR), but not here?